metron-interface/metron-rest/README.md

```diff
@@ -353,6 +355,20 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
       * searchRequest - Search request
   * Returns:
     * 200 - Search results
+    
```

Comment:
We may want to consider explaining the types we support and return here... but I am not sure it should be inline.
Since the detail is in line with the rest of the document, maybe that can be a follow on